vector/src/main/res/values/strings.xml

```diff
@@ -2521,8 +2521,8 @@
     <string name="ftue_auth_carousel_1_body">End-to-end encrypted messaging for secure and independent communication, connected via Matrix.</string>
     <string name="ftue_auth_carousel_2_title">You\'re in control.</string>
     <string name="ftue_auth_carousel_2_body">Element lets you choose where you messages are stored, keeping you in control of your data.</string>
-    <string name="ftue_auth_carousel_3_title">Connect with anyone.</string>
-    <string name="ftue_auth_carousel_3_body">Element works with all Matrix-based apps and can even bridge into proprietary messengers.</string>
+    <string name="ftue_auth_carousel_3_title">Secure Messaging.</string>
+    <string name="ftue_auth_carousel_3_body">No phone number required, so you don\'t have to share those details with the outside world. No ads, no tracking or datamining.</string>
```

Comment:
Is "no tracking" true regarding analytics? CC @daniellekirkwood

Reply:
👀  Good catch, @bmarty 

Even though we don't track without explicit consent from the user we'd like to change the content to remove the reference to tracking, that way it's super clear. 

"No ads, no datamining."

@ouchadam Is that ok?

Reply:
sounds good to me :+1:

Reply:
Thanks!